### ISSUE TITLE
Display doesn't look right (search field doesn't show)

### ISSUE THREAD

**Issue by cooljeanius** [cooljeanius/my_tweetback_archive#1]
When I serve it from my local machine via `npm start` from [its parent directory](https://github.com/cooljeanius/tweetback), everything looks fine, but the display via GitHub pages looks wrong, though: https://cooljeanius.github.io/my_tweetback_archive/ 
This has the side effect of the search field being unavailable for use.

**Comment by cooljeanius** [comment]
so according to @AramZS this is probably an instance of tweetback/tweetback#45; I'm still investigating, but ran into cooljeanius/tweetback#1 while trying to rebuild...

**Comment by katerberg** [comment]
This does seem correct that it's from https://github.com/tweetback/tweetback/issues/45. Just sharing for posterity, the way I verified this was by seeing all of the `404` errors from the CSS that did not load.

<img width="854" alt="Screenshot 2022-12-28 at 1 04 11 PM" src="https://user-images.githubusercontent.com/479643/209860350-d5260ebb-3032-484b-95c8-cfa701f39080.png">

**Comment by cooljeanius** [comment]
> This does seem correct that it's from [tweetback/tweetback#45](https://github.com/tweetback/tweetback/issues/45). Just sharing for posterity, the way I verified this was by seeing all of the `404` errors from the CSS that did not load.
> 
> <img alt="Screenshot 2022-12-28 at 1 04 11 PM" width="854" src="https://user-images.githubusercontent.com/479643/209860350-d5260ebb-3032-484b-95c8-cfa701f39080.png">

Coming back to this after a bit, and kind of forgetting where I left off... in the linked issue (45), @jakejarvis says that all that's needed for a fix is some changes to `eleventy.config.js`... but my version of that file already has the lines in question though? Well, except in my case the subdirectory is `/_site/` rather than `/tweetback/`... anyways I'm trying to rebuild, but am running into cooljeanius/tweetback#1 again... let me see if the suggestion added there helps at all...

**Comment by cooljeanius** [comment]
Darn, I thought d6319f5 would fix this (commit is too big for GitHub to render properly, apparently), but it looks like it doesn't...

**Comment by cooljeanius** [comment]
> > This does seem correct that it's from [tweetback/tweetback#45](https://github.com/tweetback/tweetback/issues/45). Just sharing for posterity, the way I verified this was by seeing all of the `404` errors from the CSS that did not load.
> > <img alt="Screenshot 2022-12-28 at 1 04 11 PM" width="854" src="https://user-images.githubusercontent.com/479643/209860350-d5260ebb-3032-484b-95c8-cfa701f39080.png">
> 
> Coming back to this after a bit, and kind of forgetting where I left off... in the linked issue (45), @jakejarvis says that all that's needed for a fix is some changes to `eleventy.config.js`... but my version of that file already has the lines in question though? Well, except in my case the subdirectory is `/_site/` rather than `/tweetback/`... anyways I'm trying to rebuild, but am running into [cooljeanius/tweetback#1](https://github.com/cooljeanius/tweetback/issues/1) again... let me see if the suggestion added there helps at all...

Hm, so @zachleat closed tweetback/tweetback#45 as a dup of tweetback/tweetback#13, but I'm still seeing this issue even after e1dfec7, though...

**Comment by cooljeanius** [comment]
progress on this issue is now being blocked by tweetback/tweetback#94 ...

**Comment by cooljeanius** [comment]
> progress on this issue is now being blocked by [tweetback/tweetback#94](https://github.com/tweetback/tweetback/issues/94) ...

ok so I worked past this; the original issue here still occurs even after a47db71 though...

**Comment by cooljeanius** [comment]
@stefanbohacek any help?

**Comment by stefanbohacek** [comment]
@cooljeanius It might be best to start fresh.

I cloned [tweetback/tweetback](https://github.com/tweetback/tweetback), followed the instructions on how to set things up, and run into the same issue with images not loading when the site was published to GitHub Pages.

Then I run:

```
npx @11ty/eleventy --quiet --pathprefix=twitterarchive
```

as per https://github.com/tweetback/tweetback/issues/13#issuecomment-1330910468.

Now the site is working fine: https://stefanbohacek.github.io/twitterarchive/

EDIT: I noticed that most of the files, including images, indeed do load, but there's a few files that are still returning 404:

- `_pagefind/pagefind-ui.js`
- `_pagefind/pagefind-ui.css`
- `_pagefind/pagefind-ui.css`

Looks like this is breaking some search functionality?

I'd try following the steps above, and if you run into the same issue, maybe you can report that?

**Comment by cooljeanius** [comment]
> @cooljeanius It might be best to start fresh.
> 
> I cloned [tweetback/tweetback](https://github.com/tweetback/tweetback), followed the instructions on how to set things up, and run into the same issue with images not loading when the site was published to GitHub Pages.
> 
> Then I run:
> 
> ```
> npx @11ty/eleventy --quiet --pathprefix=twitterarchive
> ```
> 
> as per [tweetback/tweetback#13 (comment)](https://github.com/tweetback/tweetback/issues/13#issuecomment-1330910468).
> 
> Now the site is working fine: [stefanbohacek.github.io/twitterarchive](https://stefanbohacek.github.io/twitterarchive/)
> 
> EDIT: I noticed that most of the files, including images, indeed do load, but there's a few files that are still returning 404:
> 
>    * `_pagefind/pagefind-ui.js`
> 
>    * `_pagefind/pagefind-ui.css`
> 
>    * `_pagefind/pagefind-ui.css`
> 
> 
> Looks like this is breaking some search functionality?
> 
> I'd try following the steps above, and if you run into the same issue, maybe you can report that?

OK, I tried this advice (albeit with replacing "twitterarchive" with "my_tweetback_archive"), ran into cooljeanius/tweetback#1, tried again with the fix mentioned there, and then ran into: 
```
[11ty] Writing _site/newest/index.html from ./newest.11ty.js
[11ty] Problem writing Eleventy templates: (more in DEBUG output)
[11ty] 1. Having trouble writing to "_site/1706462051126652972/index.html" from "./tweet-pages.11ty.js" (via EleventyTemplateError)
[11ty] 2. EMFILE: too many open files, open '_site/1403032746461741062/index.html' (via Error)
[11ty] 
[11ty] Original error stack trace: Error: EMFILE: too many open files, open '_site/1403032746461741062/index.html'
[11ty] Benchmark 908396ms  31% 75124× (Configuration) "htmlBaseWithPathPrefix" Transform
[11ty] Copied 8246 files / Wrote 0 files in 2947.44 seconds (v2.0.0-canary.23)
```

**Comment by cooljeanius** [comment]
ok, so after [3097885](https://github.com/cooljeanius/my_tweetback_archive/commit/3097885c3a16d00d0f136765e617b1d9668e0d52), things now at least *look* correct, but I'm still not seeing the search interface showing up, though...

**Comment by cooljeanius** [comment]
reminder to myself that if/when I attempt rebuilding this next, I need to do so from the `egallager_main` branch of [cooljeanius/tweetback](https://github.com/cooljeanius/tweetback) (i.e. *not* the `main` one), or else it won't use the correct metadata: https://github.com/cooljeanius/tweetback/blob/egallager_main/_data/metadata.js

**Comment by cooljeanius** [comment]
Update: things look even better now after [b439311](https://github.com/cooljeanius/my_tweetback_archive/commit/b439311501c673adb0c9d2d864252a188bb957f7), but the search field still isn't showing up for me yet, though...

**Comment by stefanbohacek** [comment]
@cooljeanius Yeah, I didn't get past that part either, might be worth reporting this to the project's creator as a new issue.

**Comment by cooljeanius** [comment]
> @cooljeanius Yeah, I didn't get past that part either, might be worth reporting this to the project's creator as a new issue.

OK, I opened tweetback/tweetback#96

**Comment by cooljeanius** [comment]
Seems to be fixed by e070f7b852057eac7dc6a392a6b4047438c2dbc6! Now time to update my links everywhere...
